Dockerfile

```diff
@@ -3,13 +3,15 @@ FROM debian:stable
 MAINTAINER Clifton Barnes <clifton.a.barnes@gmail.com>
 
 RUN apt-get update
-RUN apt-get install -y python python-dev python-pip lighttpd
+RUN apt-get install -y python python-dev python-pip python-smbus lighttpd
```

Comment:
The Adafruit [README](https://github.com/adafruit/Adafruit_Python_GPIO/blob/master/README.md)